Sources/FacebookCoreTests/Helpers/LocalizedStringValidation.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+//
+// You are hereby granted a non-exclusive, worldwide, royalty-free license to use,
+// copy, modify, and distribute this software in source code or binary form for use
+// in connection with the web services and APIs provided by Facebook.
+//
+// As with any software that integrates with the Facebook platform, your use of
+// this software is subject to the Facebook Developer Principles and Policies
+// [http://developers.facebook.com/policy/]. This copyright notice shall be
+// included in all copies or substantial portions of the software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+@testable import FacebookCore
+import XCTest
+
+// Uses an arbitrary class that is located in the SDK's bundle
+private let bundle = Bundle(for: AccessTokenWallet.self)
+
+func validateLocalizedStrings<T: CaseIterable & RawRepresentable>(stringsEnum: T.Type, _ file: StaticString = #file, _ line: UInt = #line) where T.RawValue == String {
+  // make sure there is entry for each case in a localizable type
```

Comment:
Oh good! I see things like this in production apps far too often (this is Netflix):

<img width="381" alt="Screen Shot 2019-03-14 at 10 28 48 AM" src="https://user-images.githubusercontent.com/6910829/54378298-3c9aa900-4644-11e9-87c3-58b8e16af801.png">